src/modules/dpos/validator/components/DelegatesTable/delegateRow.js

```diff
@@ -14,7 +14,7 @@ const DelegateRow = ({
   data, className, t, activeTab, watchList, setActiveTab, blocks,
 }) => {
   const formattedForgingTime = getForgingTime(
-    data.nextForgingTime || blocks.forgers[blocks.indexBook[data.address]]?.nextForgingTime,
+    data.nextForgingTime || blocks.forgers?.[blocks.indexBook?.[data.address]]?.nextForgingTime,
```

Comment:
@eniolam1000752 The `forgers` property might not be part of the blocks object anymore and this should be adjusted accordingly